Wiley Online Library.js

```diff
@@ -374,13 +375,22 @@ function scrapeBibTeX(doc, url) {
 			doiURLRegex = /^https:\/\/doi.org\/(.*)/;
 			if (item.DOI && item.DOI.match(doiURLRegex))
 				item.DOI = item.DOI.replace(/^https:\/\/doi.org\/(.*)/, "$1");
+			freeAccessTag(doc, item);
 			item.complete();
 		});
 
 		translator.translate();
 	});
 }
 
+//ubtue:tag an article as open access
+function freeAccessTag(doc, item) {
```

Comment:
Since the convention is that a function name describes what the function does `addFreeAccessTag()`might be a more appropriate name.

Reply:
thanks!